docs/sources/flow/_index.md

```diff
@@ -78,18 +72,3 @@ prometheus.remote_write "default" {
 [configuration language]: {{< relref "./config-language/" >}}
 [reference documentation]: {{< relref "./reference" >}}
 
-## Current limitations
-
-The goal of Grafana Agent Flow is to eventually support the same use cases that
-Grafana Agent does today. Some functionality may be missing while Grafana Agent
-Flow is still in development:
-
-* An equivalent list of integrations
-* An equivalent to the scraping service
```

Comment:
These limitations still currently exist, but since we're removing the beta status, it's probably appropriate to move to timeless documentation.